### PR TITLE
DIV-5454: Adding bailiff certificate of service template

### PR DESCRIPTION
**Pull request by jakub-kozlowski**
# Description

Added bailiff certificate of service template.

Fixes https://tools.hmcts.net/jira/browse/DIV-5454

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by shaunganley — DISMISSED** [review]

File: src/integrationTest/resources/documentgenerator/documents/jsoninput/Bailiff-certificate-of-service.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                    "poBox": "PO Box 12706",
+                    "town": "Harlow",
+                    "postcode": "CM20 9QT",
+                    "emailAddress": "divorcecase@justice.gov.uk",
+                    "phoneNumber": "0300 303 0642"
+                },
+                "caseReference": "LV17D80102",
+                "petitionerFullName": "Applicant Baker",
+                "respondentFullName": "Respondent Baker",
+                "coRespondentFullName": "Co-Respondent Baker"
```
Comment (yarpo) [inline src/integrationTest/resources/documentgenerator/documents/jsoninput/Bailiff-certificate-of-service.json:19]: please update the template `FL-DIV-GNO-ENG-00595.docx` to use boolean `hasCoRespondent`.

And then, please add 2 scenarios here: for `true` and `false`.

**Review by johnoneill46 — DISMISSED** [review]

File: src/integrationTest/resources/documentgenerator/documents/jsoninput/Bailiff-certificate-of-service-with-corespondent.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                    "poBox": "PO Box 12706",
+                    "town": "Harlow",
+                    "postcode": "CM20 9QT",
+                    "emailAddress": "divorcecase@justice.gov.uk",
+                    "phoneNumber": "0300 303 0642"
+                },
+                "caseReference": "LV17D80102",
+                "petitionerFullName": "Applicant Baker",
+                "respondentFullName": "Respondent Baker",
+                "hasCoRespondent": "true",
```
Comment (yarpo) [inline src/integrationTest/resources/documentgenerator/documents/jsoninput/Bailiff-certificate-of-service-with-corespondent.json:19]:
```
"hasCoRespondent": true,
```

valid values for JSON:
image

Boolean is boolean, "true" / "false" may be (and should be) treated as a string :)

Comment (yarpo) [reply src/integrationTest/resources/documentgenerator/documents/jsoninput/Bailiff-certificate-of-service-with-corespondent.json:19]: source: https://www.json.org/json-en.html

**Review by HarryH96 — APPROVED** [review]

**Review by yarpo — APPROVED** [review]
<3

**Review by ocarleton — APPROVED** [review]

**Pull request merged**